finisher_api.go

```diff
@@ -75,7 +75,12 @@ func (db *DB) Save(value interface{}) (tx *DB) {
 
 	reflectValue := reflect.Indirect(reflect.ValueOf(value))
 	for reflectValue.Kind() == reflect.Ptr || reflectValue.Kind() == reflect.Interface {
-		reflectValue = reflect.Indirect(reflectValue)
```

Comment:
`reflect.Indirect` returns itself if it is not a pointer, just remove `reflectValue.Kind() == reflect.Interface` .
Please add unit tests to describe it.